lib/AcqKeyboardShortcutsModal/AcqKeyboardShortcutsModal.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { KeyboardShortcutsModal } from '@folio/stripes/components';
+
+import { acqCommands } from '../constants';
+
+const AcqKeyboardShortcutsModal = ({
+  commands = [],
+  onClose,
+}) => (
+  <KeyboardShortcutsModal
+    open
```

Comment:
Maybe propagate the `open` prop?
So instead of 
```jsx
      {isOpen && (
        <AcqKeyboardShortcutsModal
          onClose={toggleModal}
        />
      )}
```
we could use it like this
```jsx
        <AcqKeyboardShortcutsModal
          open={isOpen}
          onClose={toggleModal}
        />
```